.gitignore

```diff
@@ -60,3 +60,6 @@ target/
 
 #Ipython Notebook
```

Comment:
Not really review comment as such, but I should probably learn ipython at some point...